Lib/socketserver.py

```diff
@@ -230,6 +230,10 @@ def serve_forever(self, poll_interval=0.5):
 
                 while not self.__shutdown_request:
                     ready = selector.select(poll_interval)
+                    # bpo-35017: request arriving after the server shutdown
```

Comment:
I'm confused by this comment. I suggest something like:

"bpo-35017: shutdown() has been called from a different thread while select() was polling. Don't handle the request, exit immediatly."

... or maybe remove the comment, I don't know :-)

Reply:
If I may suggest a shorter version with the same meaning:
```python
# bpo-35017: shutdown() called during select(), exit immediately
``` 
Though I think the code is quite self-explanatory, and I would omit the comment ;-)

Reply:
Let me know If I should use @odony  reword, or if you prefer removing the comment completely.

Reply:
"# bpo-35017: shutdown() called during select(), exit immediately" is better than my long boring comment :)

Reply:
@vstinner Off topic but just as a pro-tip now you can suggest changes in GitHub UI that can be accepted and committed from UI when the contributor clicks "Accept suggestion" that might help during rewording :) Ref : https://twitter.com/github/status/1054794897448939520 

```suggestion
                    # bpo-35017: shutdown() called during select(), exit immediately
```